packages/tasks/src/tasks/zero-shot-image-classification/about.md

```diff
@@ -32,7 +32,8 @@ The model can be loaded with the zero-shot-image-classification pipeline like so
 from transformers import pipeline
 # More models in the model hub.
 model_name = "openai/clip-vit-large-patch14-336"
-classifier = pipeline("zero-shot-image-classification", model = model_name)
+device_name = 0
+classifier = pipeline("zero-shot-image-classification", model = model_name, device = device_name)
```

Comment:
```suggestion
classifier = pipeline("zero-shot-image-classification", model=model_name, device=0)
```

(let's not use `device_name` for a number)

Reply:
True. I can do Cuda:0. I will wait for Merve's answer.

Reply:
I think it would be even better if you could do `device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')` and pass device